### PR TITLE
Added Swahili translation for the administrate file

### PR DESCRIPTION
**Pull request by larymak**
Added a Swahili translation of the terrastories per the directions given in issue no: [826](https://github.com/Terrastories/terrastories/issues/826). 

To begin with, I translated the `adminstarte.sw.yml` file, currently working on the other files. I would like to be a long-term contributor fo the project.

### CODE REVIEW

**Comment by julinvictus** [comment]
Hi @larymak, please add only the file(s) you have translated to this PR, it's fine to add the other ones later.

**Comment by larymak** [comment]
The file I have translated is added to the PR

**Review by rudokemper — APPROVED** [review]
Hello! I think it's fine to have the other files left in English for now, since without them, we will have a lot of `translation missing` messages and English is a good fallback for Swahili.

Thanks so much @larymak! There are communities like the Ogiek in Kenya interested in using Terrastories so this is awesome to have. Appreciate your support and interest in Terrastories :)

**Comment by rudokemper** [comment]
Apologies, I am reverting on the review approval because I think we should have some of the core content for the front-end in `sw.yml` in place first, to @julinvictus's point. Because right now if we merge this, Swahili will show up as an option, but not accessible anywhere on the app except on the Administrate views.

I will revise the Issue to reflect this MVP need for merging of translation PRs.

File: rails/config/locales/sw/dashboard.sw.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+en:
```
Comment (lauramosher) [inline rails/config/locales/sw/dashboard.sw.yml:1]: The file name has been updated to use `.sw` but the translation file key is still keyed to english translations. Let's update the file to denote "sw":

```suggestion
sw:
```

Even if the translations in the file are still English, the top-level key dictates the locale lookup and must be updated to `sw`.

**Review by lauramosher — DISMISSED** [review]
Hello! Thanks for your contribution to adding swahili to Terrastories!

While maintaining English translations for the keys is totally fine as placeholders, the files do need to be namespaced for `sw` and not `en` — renaming the file itself is not enough for translations to work across the app.

File: rails/config/locales/sw/dashboard.sw.yml (hunk 1 above)
Comment (larymak) [reply rails/config/locales/sw/dashboard.sw.yml:1]: I am working on it and will update the PR once I have completed the translation

Comment (larymak) [reply rails/config/locales/sw/dashboard.sw.yml:1]: I have translated and updated the `sw.yml` file

Comment (rudokemper) [inline rails/config/locales/sw/dashboard.sw.yml:1]: We just have to update this to `sw` in all of the remaining files that are still in English, to be able to merge this PR :)

Comment (larymak) [reply rails/config/locales/sw/dashboard.sw.yml:1]: Is there anything needed from my end?

Comment (rudokemper) [reply rails/config/locales/sw/dashboard.sw.yml:1]: Yes, please update `en`  to `sw` at the beginning of each of the remaining files:

* dashboard.sw.yml
* devise.sw.yml
* models.sw.yml

Comment (larymak) [reply rails/config/locales/sw/dashboard.sw.yml]: I have updated the 3 files.

**Review by rudokemper — APPROVED** [review]
Looks great. Thanks for your work on this @larymak !

**Pull request merged**